pkg/devfile/parser/data/v2/2.2.0/devfileJsonSchema220.go

```diff
@@ -347,7 +347,7 @@ const JsonSchema220 = `{
                     },
                     "name": {
                       "type": "string",
-                      "maxLength": 63,
+                      "maxLength": 15,
```

Comment:
do we have corresponding updates in our API docs to show the new max length?

Reply:
the spec is the api doc